src/Logging/Logging.Console/src/ConsoleLoggerProcessor.cs

```diff
@@ -49,18 +49,24 @@ internal virtual void WriteMessage(LogMessageEntry message)
         {
             var console = message.LogAsError ? ErrorConsole : Console;
 
-            if (message.TimeStamp != null)
+            try
```

Comment:
Not sure we need to wrap this, `ProcessLogQueue` already has a try catch and if the console throws we complete the blockingcollection since there was a problem.

And the only other call to `WriteMessage` is directly from the console logger. We might still want to avoid throwing directly from the console logger too, in that case we should move this try catch to `EnqueueMessage`.